al-khaser/Anti Debug/NtQueryObject_AllTypesInformation.cpp

```diff
@@ -10,6 +10,8 @@ locate "DebugObject". Todo: Support for Win10
 
 BOOL NtQueryObject_ObjectAllTypesInformation()
 {
+	//NOTE this check is unreliable, a debugger present on the system doesn't mean it's attached to you
+

```

Comment:
ahah all good :) I got it. It is ok for me. If someone test functions separately, maybe then it would have sense.